src/components/Table.tsx

```diff
@@ -439,7 +438,7 @@ export default class Table<T> extends React.Component<
 		const sortedData = this.sortData(data).slice(lowerBound, upperBound);
 
 		return (
-			<Box {...props}>
+			<React.Fragment>
```

Comment:
Is there a reason we're not using the built-in `<> ... </>` syntax for this? https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html. Should be equivalent, and it's supported in React 16.2+ while it looks like we're using 16.3.1.

Reply:
Was used to write it that way, nothing more. Will update it 👍